claat/nodes/survey_test.go

```diff
@@ -0,0 +1,210 @@
+package nodes
+
+import (
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+)
+
+func TestNewSurveyNode(t *testing.T) {
+	tests := []struct {
+		name     string
+		inID     string
```

Comment:
Consider dropping the `in` prefix; `id` and `groups` are sufficient.

Reply:
Discussion on similar comment https://github.com/googlecodelabs/tools/pull/594/files/8acfb33186a4e580c372a34d41087c721343dd3d#r666514627